packages/ecs-morgan-format/CHANGELOG.md

```diff
@@ -8,6 +8,26 @@
 
 - Set `http.request.id` field (see [ecs-helpers CHANGELOG](../ecs-helpers/CHANGELOG.md#v210)).
 
+- Changed to a named export. The preferred way to import is now:
+
+  ```js
+  const { ecsFormat } = require('@elastic/ecs-morgan-format'); // CommonJS
+  import { ecsFormat } from '@elastic/ecs-morgan-format'; // ESM
+  ```
+
+  The old way will be deprecated and removed in the future:
+
+  ```js
+  const ecsFormat = require('@elastic/ecs-morgan-format'); // OLD
+  ```
+
+  Changes in this version add support for default import in TypeScript, with or
+  without the `esModuleInterop` setting:
+
+  ```ts
+  import ecsFormat from '@elastic/ecs-morgan-format';
```

Comment:
if I'm not mistaken this is a default import right? should we recommend to use named ones?

Reply:
Yes, that is a default import. I suppose we could just not mention this change. Before this PR, doing that type of import from **TYpeScript** and without `esModuleInterop: true` in tsconfig.json would fail. That works now... but in the same change we prefer people use named imports.